calib_file.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pandas as pd
+
+class CalibFile:
+    '''
+    CalibFile stores calibration points.
+
+    The data is stored in a pandas data frame for easy r/w. 90% of this class
+    is just properties so the rows and columns can be r/w'ed in python as if 
+    they were just normal variables.
+
+    Properties
+    ----------
+
+    start_pt : pd.Series
+        Vector storing the location of the first calibration point. Can be set
+        with np array or pd series indexed ['x','y','z'].
+
+    n_pt : pd.Series
+        Vector storing the location of the calibration point on the N axis. Can
+        be set with np array or pd series indexed ['x','y','z'].
+
+
+    m_pt : pd.Series
+        Vector storing the location of the calibration point on the M axis. Can
+        be set with np array or pd series indexed ['x','y','z'].
+
+
+    N : int
+        number of samples traversed from start_pt to reach n_pt
+
+    M : int
+        number of samples traversed from start_pt to reach m_pt
+
+    '''
+    def __init__(self,from_file_name=None):
+        '''
+        Create an empty calibration instance or construct one from the given
+        file.
+        '''
+        self.axis = ['x','y','z']
+        if from_file_name != None:
```

Comment:
`if from_file_name:` or (if you explicitly need to check for `None`) `if from_file_name is not None:`